libpretixui-android/src/main/java/eu/pretix/libpretixui/android/questions/QuestionsDialog.kt

```diff
@@ -651,7 +646,7 @@ class QuestionsDialog(
 
     class QuestionInvalid(val msgid: Int) : Exception()
 
-    private fun serializeAnswer(question: QuestionLike): Answer {
+    private fun serializeAnswer(question: QuestionLike, requiredOverride: Boolean): Answer {
```

Comment:
Same comment as in libpretixsync, I don't think this is a good name. Also, this PR even includes two different names?

Reply:
yeah, will go with `allOptional` for this param. `answersAreOptionalOverride` is weird too, maybe drop the override and go with `allAnswersAreOptional` for the class-wide var?

Reply:
it's `allAnswersAreOptional` now :)